<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor XXXXX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ActorImageView" name="actorImgeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="ActorFrameView" name="actorAnimationView">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>30</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>580</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="AnimationFrameTree" name="animationTreeWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>130</number>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">Cell</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Position</string>
     </property>
    </column>
   </widget>
   <widget class="ActorBuildView" name="actorBuildView">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResetOffset">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>190</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxOffsetX">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxOffsetY">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxFrameCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="AnimationSelectList" name="animationSelectListWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAddAnimation">
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActorFrameView</class>
   <extends>QGraphicsView</extends>
   <header>actorframeview.h</header>
  </customwidget>
  <customwidget>
   <class>ActorBuildView</class>
   <extends>QGraphicsView</extends>
   <header>actorbuildview.h</header>
  </customwidget>
  <customwidget>
   <class>AnimationFrameTree</class>
   <extends>QTreeWidget</extends>
   <header>animationframetree.h</header>
  </customwidget>
  <customwidget>
   <class>ActorImageView</class>
   <extends>QGraphicsView</extends>
   <header>actorimageview.h</header>
  </customwidget>
  <customwidget>
   <class>AnimationSelectList</class>
   <extends>QListWidget</extends>
   <header>animationselectlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
